src/cluster/Cargo.toml

```diff
@@ -16,7 +16,6 @@ cli = [
     "structopt",
     "color-eyre",
     "prettytable-rs",
-    "fluvio-runner-local/platform",
     "fluvio-extension-common/target",
 ]
 
```

Comment:
does this need version bump?

Reply:
I don't think so, because the API of the cluster crate has stayed the same